### PR TITLE
[ADD] NER

### PR DESCRIPTION
**Pull request by sihrc**

### CODE REVIEW

File: indicoio/text/ner.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    languages of what language the text was written in.
+
+    Example usage:
+
+    .. code-block:: python
+
+       >>> import indicoio
+       >>> import numpy as np
+       >>> text = 'Monday: Delightful with mostly sunny skies. Highs in the low 70s.'
+       >>> possible = indicoio.language(text)
```
Comment (anneCarlson) [inline indicoio/text/ner.py]: The example usage is for `language`. I bet that is the case other places as well.

Comment (sihrc) [inline indicoio/text/ner.py]: :+1: haven't added tests / examples yet

**Comment by madisonmay** [comment]
One final bit of code review before merge to development @anneCarlson?

File: indicoio/__init__.py [hunk 2]
```diff
@@ -25,5 +26,6 @@
 apis = dict((api, globals().get(api)) for api in API_NAMES)
 
 for api in apis:
+    print api
```
Comment (madisonmay) [inline indicoio/__init__.py]: Doh -- stray print statement.  Fixing...

**Comment by anneCarlson** [comment]
Looks good to me!

**Comment by madisonmay** [comment]
Just rebased things, should merge cleanly now.

**Pull request merged**